.clusterfuzzlite/project.yaml

```diff
@@ -0,0 +1,9 @@
+homepage: https://sigstore.dev/
+language: go 
```

Comment:
note that you only need this line and not the others for CFLite (unlike in OSS-Fuzz where you do need them).

Reply:
Took care of it. Thanks